package.json

```diff
@@ -83,6 +82,8 @@
   },
   "devDependencies": {
     "@babel/core": "7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
```

Comment:
These packages appear to be deprecated <https://www.npmjs.com/package/@babel/plugin-proposal-class-properties>, <https://www.npmjs.com/package/@babel/plugin-proposal-optional-chaining>

Reply:
Why are these needed now? How did we get this far without them?

Reply:
These two are required by the react-helmet